decentraland/unity-renderer
# feat: fixed text models to be equals to their kernel counterpart

## What does this PR change?

Fixes #805

Before this PR , the TextShape, UIText and UIInputText shared the same text model, but in kernel they have differents attributes for each model. With this PR, now they have the same model as it is implemented in kernel. However, there is some attributes that we are not using in Unity

## How to test the changes?


1. Go to: https://play.decentraland.zone/index.html?renderer=urn:decentraland:off-chain:renderer-artifacts:feat/update-text-model
2. Go to a place where a UIText, UIInputText and TextShape exists  and check that everything work as expected

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
